### ISSUE TITLE
Please format Status and Created to time instead of strings

### ISSUE THREAD

**Issue by beaudryj** [samneirinck/posh-docker#22]
This really sucks for Using where object against

**Comment by samneirinck** [comment]
Could you clarify a bit? Are you talking about the normal `docker` commands? Or are you piping its output to `ConvertFrom-Docker`?

**Comment by beaudryj** [comment]
Yes sorry I didn't mean to leave that so cryptic haha, don't work on 8
things at once :p. But yes when using convert from-docker it would be
really since if we could native call status or created as time objects vs
strings
